### ISSUE TITLE
Extension Details - Contributions tab

### ISSUE THREAD

**Issue by wsmelton** [Microsoft/vscode#47586]
- VSCode Version: N/A
- OS Version: N/A

Steps to Reproduce:

1. Open VS Code
2. Open up the view/tab for any extension you have installed
3. Click on the "Contributions" tab

![image](https://user-images.githubusercontent.com/11204251/38558973-636cd8f8-3c97-11e8-9447-c25ad7c413ba.png)

This is just cosmetics but being that this is called "Contributions" I'm assuming "documentation" does not go in this tab for an extension? I checked other random ones I have installed and ones I don't and it seems the average information listed here is the key bindings. Which I am assuming most have been doing because all of the Microsoft authored extensions I find to put the same type of information in this tab.

Can it just be renamed to `Customizations` or something more description to what it is since we are not using it for what is called now? It would be beneficial to have that information on any given extension for discovery. It would possibly encourage authors to document those options more easily; one of the main things I have to go hunting for on extensions that don't.

**Comment by usernamehw** [comment]
The content of `contributions` is generated by vscode automatically. It's handy when you want to see what extension is adding to your editor.

The docs should go into the `details` tab.

Maybe adding a `title` attribute for each(4) tab name would clarify things for everyone...

**Comment by sandy081** [comment]
`Contributions` tab shows those an extension contributes to VS Code. Customizations might mislead to settings and keybindings. It is more than that.

Adding `title` attribute is good.

**Comment by bryce-s** [comment]
#47803 
I gave this a shot! Titles in the PR are as follows:

``Details`` -> ``Extension details``
``Contributions`` -> ``Extension modifications``
``Changelog`` -> ``Extension change history``
``Dependencies`` -> ``Extension dependencies``

**Comment by InspectorDeno** [comment]
#49825
With the tooltip titles:
`Details` -> `Extension details`
`Contributions` -> `Extension contributions to VS Code editor`
`Changelog` -> `Extension changelog`
`Dependencies` -> `Extension dependencies`

**Comment by isidorn** [comment]
The current tooltip does not really work for me "VS Code editor" some extensions can contribute to things which are not the editor.
Also in explaining the term contributions I would not use the word contribution. Try something else

![screen shot 2018-05-29 at 14 56 04](https://user-images.githubusercontent.com/1926584/40660163-7c62bef6-6350-11e8-9c53-66adaaf16cf8.png)

**Comment by sandy081** [comment]
@isidorn Here, VS Code editor represents the complete editor.. not just the editor part. If I am right, Even in API we call complete product - `editor`. 

Regarding `contributions` I am happy to accept suggestions. Just note that the tab name is called Contributions.

**Comment by isidorn** [comment]
Yes I understand that. However in our documentation and release notes we always just say VS Code. imho that is clearer.
`Lists functionality added by this extension to VS Code.`
